rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessParamTagRector/Fixture/x.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessParamTagRector\Fixture;
+
+class X {
+	/**
+	 * @param Closure(int): bool $filter
```

Comment:
it possibly need to have `\` prefix as inside namespace, or add use statement for `Closure`: 

```suggestion
	 * @param \Closure(int): bool $filter
```